lms/djangoapps/certificates/api.py

```diff
@@ -63,6 +63,7 @@ def format_certificate_for_user(username, cert):
         "grade": cert.grade,
         "created": cert.created_date,
         "modified": cert.modified_date,
+        "passing": is_passing_status(cert.status),
```

Comment:
`passed` makes me think of dates passing, not grades.  But I'm ok with `is_passing`.  Fixed in 57db57e.